recipes/tk/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "8.5.19" %}
+{% set maj_min = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: tk
+  version: {{ version }}
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python         # [win]
+    - curl           # [win]
+    - 7za            # [win]
+
+test:
+  files:
+    - hello.tcl
+
+  commands:
+    # Check for binaries.
+    - test -f "${PREFIX}/bin/tclsh"                      # [unix]
+    - test -f "${PREFIX}/bin/tclsh{{ maj_min }}"         # [unix]
+    - test -f "${PREFIX}/bin/wish"                       # [unix]
+    - test -f "${PREFIX}/bin/wish{{ maj_min }}"          # [unix]
+
+    # Check for includes.
+    - test -f "${PREFIX}/include/tclDecls.h"             # [unix]
+    - test -f "${PREFIX}/include/tcl.h"                  # [unix]
+    - test -f "${PREFIX}/include/tclPlatDecls.h"         # [unix]
+    - test -f "${PREFIX}/include/tclTomMathDecls.h"      # [unix]
+    - test -f "${PREFIX}/include/tclTomMath.h"           # [unix]
+    - test -f "${PREFIX}/include/tkDecls.h"              # [unix]
+    - test -f "${PREFIX}/include/tk.h"                   # [unix]
+    - test -f "${PREFIX}/include/tkPlatDecls.h"          # [unix]
+
+    # Check for libraries.
+    - test -f "${PREFIX}/lib/libtclstub{{ maj_min }}.a"  # [unix]
+    - test -f "${PREFIX}/lib/libtkstub{{ maj_min }}.a"   # [unix]
+    - test -f "${PREFIX}/lib/libtcl{{ maj_min }}.so"     # [linux]
+    - test -f "${PREFIX}/lib/libtcl{{ maj_min }}.dylib"  # [osx]
+
+    # Check commands work and run a simple program.
+    - tclsh{{ maj_min }} hello.tcl
+    - tclsh hello.tcl
+    - wish{{ maj_min }} hello.tcl
+    - wish hello.tcl
+
+about:
+  home: http://www.tcl.tk/
+  license: Tcl/Tk
+  license_family: BSD
+  summary: A dynamic programming language with GUI support.
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - jjhelmus
+    - msarahan
+    - pelson
```

Comment:
@jjhelmus @ocefpaf, if you guys decide you don't want to be on this for some reason, please let me know and I will remove you. Though we would be much happier if you stayed. 😄